webhook/resourcesemantics/defaulting/defaulting.go

```diff
@@ -231,8 +262,18 @@ func (ac *reconciler) mutate(ctx context.Context, req *admissionv1.AdmissionRequ
 	logger := logging.FromContext(ctx)
 	handler, ok := ac.handlers[gvk]
 	if !ok {
-		logger.Error("Unhandled kind: ", gvk)
-		return nil, fmt.Errorf("unhandled kind: %v", gvk)
+		if _, ok := ac.callbacks[gvk]; !ok {
+			logger.Error("Unhandled kind: ", gvk)
+			return nil, fmt.Errorf("unhandled kind: %v", gvk)
+		}
+		patches, err := ac.callback(ctx, gvk, req, duck.JSONPatch{})
```

Comment:
By doing this here we're not setting up the context correctly - ie. there's subresources, user info etc. setting up the user annotation etc.